doc/devel/style_guide.rst

```diff
@@ -0,0 +1,268 @@
+| started using Matplotlib.          | to get started using Matplotlib.   |
++------------------------------------+------------------------------------+
+|  1. Import the Matplotlib library. |  - Import the Matplotlib library.  |
+|  2. Import the necessary modules.  |  - Import the necessary modules.   |
+|  3. Set and assign data to work on.|  - Set and assign data to work on. |
+|  4. Transform data with methods and|  - Transform data with methods and |
+|     functions.                     |    functions.                      |
++------------------------------------+------------------------------------+
+
+Tables
+^^^^^^
+Use ASCII tables with reStructuredText standards in organizing content. 
+Markdown tables and the csv-table directive are not accepted.
+
++--------------------------------+----------------------------------------+
+| Correct                        | Incorrect                              |
++================================+========================================+
+| +----------+----------+        | ::                                     |
+| | Correct  | Incorrect|        |                                        |
+| +==========+==========+        |     | Correct | Incorrect |            |
+| | OK       | Not OK   |        |     | ------- | --------- |            |
+| +----------+----------+        |     | OK      | Not OK    |            |
+|                                |                                        |
++--------------------------------+----------------------------------------+
+| ::                             | ::                                     |
+|                                |                                        |
+|     +----------+----------+    |     .. csv-table::                     |
+|     | Correct  | Incorrect|    |        :header: "correct", "incorrect" |
+|     +==========+==========+    |        :widths: 10, 10                 |
+|     | OK       | Not OK   |    |                                        |
+|     +----------+----------+    |        "OK   ", "Not OK"               |
+|                                |                                        |
++--------------------------------+                                        |
+| ::                             |                                        |
+|                                |                                        |
+|     ===========  ===========   |                                        |
+|       Correct     Incorrect    |                                        |
+|     ===========  ===========   |                                        |
+|     OK           Not OK        |                                        |
+|     ===========  ===========   |                                        |
+|                                |                                        |
++--------------------------------+----------------------------------------+
+
+
+Additional Resources
+====================
+
```

Comment:
Can this get a short sentence providing some context? What should readers take from them? I don't think it's "Follow these guides as well", but rather "This style guide gives some fundamental guide lines, but cannot specify every aspect of documentation. Think of what is helpful for the reader and use common sense when writing documentation. Further helpful ideas can be found in the following guides."